web/src/main/java/org/springframework/security/web/server/authorization/ExceptionTranslationWebFilter.java

```diff
@@ -39,12 +39,12 @@
 
 	@Override
 	public Mono<Void> filter(ServerWebExchange exchange, WebFilterChain chain) {
-		return chain.filter(exchange)
```

Comment:
Please remove these changes which appear to just be formatting. If you want to clean up formatting, that can be done w/ another commit